experimental/traffic-portal/src/app/core/cache-groups/cache-group-table/cache-group-table.component.spec.ts

```diff
@@ -185,23 +185,7 @@ describe("CacheGroupTableComponent", () => {
 		expect(menuItem.href({...sampleCG, id: 5})).toBe("core/cache-groups/5");
 	});
 	it("doesn't allow selection of unimplemented context menu items", () => {
-		let menuItem = component.contextMenuItems.find(i => i.name === "Manage ASNs");
-		if (!menuItem) {
-			return fail("'Manage ASNs' context menu item not found");
-		}
-		if (!isAction(menuItem)) {
-			return fail(`Invalid 'Manage ASNs' context menu item; not an action: ${menuItem}`);
-		}
-		if (typeof(menuItem.disabled) !== "function") {
-			return fail("'Manage ASNs' context menu item should be disabled, but no disabled function is defined");
-		}
-		if (menuItem.multiRow) {
-			expect(menuItem.disabled([sampleCG])).toBeTrue();
-		} else {
-			expect(menuItem.disabled(sampleCG)).toBeTrue();
-		}
-
-		menuItem = component.contextMenuItems.find(i => i.name === "Manage Servers");
```

Comment:
Now that this is working, this spec should include a test for it instead of a test that checks that it's always disabled - which you are correctly deleting.

Reply:
I thought I should have but wasn't sure.. thanks for confirming.